tests/foreman/cli/test_errata.py

```diff
@@ -237,7 +257,86 @@ def test_negative_install_by_hc_name_without_org_info(self):
+              custom repository
+
+           note: all products repositories contain erratum
+           """
+        super(ErrataTestCase, cls).setUpClass()
+        cls.org = make_org()
+        cls.org_product = make_product(
+            options={'organization-id': cls.org['id']})
+        cls.org_custom_repo = make_repository({
+            'download-policy': 'immediate',
+            'organization-id': cls.org['id'],
+            'product-id': cls.org_product['id'],
+            'url': FAKE_6_YUM_REPO
+        })
+        Repository.synchronize({'id': cls.org_custom_repo['id']})
+        cls.org2 = make_org()
+        cls.org2_product = make_product(
+            options={'organization-id': cls.org2['id']})
+        cls.org2_custom_repo = make_repository({
+            'download-policy': 'immediate',
+            'organization-id': cls.org2['id'],
+            'product-id': cls.org2_product['id'],
+            'url': FAKE_1_YUM_REPO
+        })
+        Repository.synchronize({'id': cls.org2_custom_repo['id']})
+        cls.org3 = make_org()
+        cls.org3_product1 = make_product(
+            options={'organization-id': cls.org3['id']})
+        cls.org3_custom_repo1 = make_repository({
+            'download-policy': 'immediate',
+            'organization-id': cls.org3['id'],
+            'product-id': cls.org3_product1['id'],
+            'url': FAKE_0_YUM_REPO
+        })
+        Repository.synchronize({'id': cls.org3_custom_repo1['id']})
+        cls.org3_product2 = make_product(
+            options={'organization-id': cls.org3['id']})
+        cls.org3_custom_repo2 = make_repository({
+            'download-policy': 'immediate',
+            'organization-id': cls.org3['id'],
+            'product-id': cls.org3_product2['id'],
+            'url': FAKE_3_YUM_REPO
+        })
+        Repository.synchronize({'id': cls.org3_custom_repo2['id']})
+
+    @staticmethod
+    def _get_sorted_erratum_ids_info(erratum_ids, sort_by='issued',
+                                     sort_reversed=False):
+        """Query hammer for erratum ids info
+        :param erratum_ids: a list of errata id
+        :param sort_by: the field to sort by the results (issued or updated)
+        :param sort_reversed: whether the sort should be reversed
+               (not ascending)
+        :return: a list of errata info dict for each errata id in erratum_ids
+        :type erratum_ids: list[str]
+        :type sort_by: str
+        :type sort_reversed: bool
+        :rtype: list[dict]
+        """
+        if len(erratum_ids) > ERRATUM_MAX_IDS_INFO:
+            raise Exception('Erratum ids length exceeded')
+        erratum_info_list = []
+        for errata_id in erratum_ids:
+            erratum_info_list.append(
+                Erratum.info(options={'id': errata_id}, output_format='json'))
+        sorted_erratum_info_list = sorted(
+            erratum_info_list, key=itemgetter(sort_by), reverse=sort_reversed)
```

Comment:
if you do not use itemgetter you have to use lamda function to retrieve the values from the dict be compared by sorted function